assignment/core/src/main/java/com/assignment/core/schedulers/JobScheduler.java

```diff
@@ -0,0 +1,55 @@
+package com.assignment.core.schedulers;
+
+import com.assignment.core.services.JobTriggerService;
+import org.apache.sling.commons.scheduler.ScheduleOptions;
+import org.apache.sling.commons.scheduler.Scheduler;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The  JobScheduler class is a scheduled task that triggers a job at a fixed interval.
+ * It implements the  Runnable interface to be used by the Sling Scheduler.
+ */
+@Component(service = Runnable.class, immediate = true)
+public class JobScheduler implements Runnable {
```

Comment:
Create configuration for this JobScheduler also don't just pass direct cron to activate scheduler.